cfme/fixtures/pytest_selenium.py

```diff
@@ -1137,7 +1150,11 @@ def _login_func():
                 'service evmserverd status').output)
             raise
 
-    if recycle:
+    if restart_evmserverd:
+        store.current_appliance.ssh_client.run_command("service evmserverd restart")
```

Comment:
Nav can handle it because when you recycle a browser, you automatically have to log in, so from the nav perspective it is no problem. This is caused by the memory leaks of CFME, so restarting should fix the issue for some time until it blows up. The nav is the only place where we can do the restart.